test/components/save_this_page_test.rb

```diff
@@ -0,0 +1,45 @@
+require "component_test_helper"
+
+class SaveThisPageTest < ComponentTestCase
+  def component_name
+    "save_this_page"
+  end
+
+  test "does not render without page path" do
+    assert_empty render_component({})
```

Comment:
Is it worth doing another assert here with attributes other than `page_path` to make sure that it's not rendering? Maybe something like:

```
assert_empty render_component({
    description: "this won't render",
    data: {
        signed_in: true,
        page_is_saved: true
    }
})
```

Reply:
That's a very good point, have updated the tests with your suggestion 👍